docs/user_manual/guides/sign_off_case.md

```diff
@@ -0,0 +1,38 @@
+# Complete Case Sign-Offs
+
+The final three QC steps are completed at the case level. These sign-offs are recorded in another
+system (Nabu), but can be completed from Dimsum.
```

Comment:
Is a connected Nabu instance a requirement to run dimsum? what happens if there's no connected Nabu?

Reply:
As far as users are concerned, it's required. It's basically required for production, as we wouldn't be able to complete these sign-offs without it.

For development purposes, it's not strictly required though. When Nabu integration is disabled (not configured), the "Sign Off" button still appears, but when attempting to save, an error is shown indicating that Nabu integration is not enabled.